test/helpers/utils.go

```diff
@@ -518,6 +520,45 @@ func MCDForNode(cs *framework.ClientSet, node *corev1.Node) (*corev1.Pod, error)
 	return mcdForNode(cs, node)
 }
 
+// Returns the node's uptime expressed as the total number of seconds the
+// system has been up.
+// See: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/s2-proc-uptime
+func GetNodeUptime(t *testing.T, cs *framework.ClientSet, node corev1.Node) float64 {
```

Comment:
We do something similar in mcd_test and sno_mcd_test. Doesn't have to be part of this PR, but we should maybe share the implementation

Reply:
100% agreed. I wrote these helpers with that in mind. Once this lands, I'll open a separate PR to fix those.